docs/jupiterone.md

```diff
@@ -91,18 +76,28 @@ https://github.com/JupiterOne/sdk/blob/master/docs/integrations/development.md
 
 The following entities are created:
 
-| Resources    | Entity `_type`     | Entity `_class`           |
-| ------------ | ------------------ | ------------------------- |
-| Organization | `tfe_organization` | `Account`, `Organization` |
-| User         | `tfe_user`         | `User`                    |
+| Resources       | Entity `_type`           | Entity `_class`           |
+| --------------- | ------------------------ | ------------------------- |
+| Account         | `tfe_account`            | `User`                    |
```

Comment:
I think the _class should be 'Account' if we're keeping this entity.

Reply:
Agree!